include/cantera/transport/DustyGasTransport.h

```diff
@@ -19,8 +19,9 @@ namespace Cantera
 {
 //! Class DustyGasTransport implements the Dusty Gas model for transport in porous media.
 /*!
- * As implemented here, only species transport is handled. The viscosity,
- * thermal conductivity, and thermal diffusion coefficients are not implemented.
+ * As implemented here, only species transport is handled. The viscosity
+ * and thermal diffusion coefficients are not implemented. Thermal conductivity
+ * is calculated as the conductivity of the gas phase.
```

Comment:
There were no changes here to the `thermalConductivity` method of the C++ `DustyGasTransport` class. That still raises a `NotImplementedError`, which I think is correct In that case, this comment was more correct as it originally was.